website/docs/docs/core/connect-data-platform/about-core-connections.md

```diff
@@ -30,3 +30,27 @@ These connection instructions provide the basic fields required for configuring
 If you're using dbt from the command line (CLI), you'll need a profiles.yml file that contains the connection details for your data platform. When you run dbt from the CLI, it reads your dbt_project.yml file to find the profile name, and then looks for a profile with the same name in your profiles.yml file. This profile contains all the information dbt needs to connect to your data platform.
 
 For detailed info, you can refer to the [Connection profiles](/docs/core/connect-data-platform/connection-profiles).
+
+
+## Adapter features
+
+The following table lists the features available for adapters:
+
+| Adapter | Catalog | Source freshness |
+|---------|---------|------------------|
+| dbt default configuration | manual run | `loaded_at` field |
+| `dbt-bigquery` | incremental | metadata-based |
```

Comment:
Anything that has `metadata-based` source freshness _also_ can configure it via the `loaded_at` field - I don't think that's currently clear

Reply:
Same note for catalog features

Reply:
@graciegoheen , clarifying question: what should be added for the catalog features?

Reply:
If an adapter supports `incremental`/`partial` catalog generation, it also support full - not actually sure what "manual run" means here?

Reply:
thanks for the clarification!